### PR TITLE
chore(mock): add mock data container

### PR DESCRIPTION
**Pull request by DanielHabenicht**

### CODE REVIEW

**Comment by Zeroks77** [comment]
@DanielHabenicht du musst @paule96 schon als Reviewer hinzufügen :)

**Comment by DanielHabenicht** [comment]
@Zeroks77 du kannst ja auch reviewen ;)

File: Phonebook.Source.MockPeopleBackend/mocks/real/persons.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        "Firstname": "Max",
+        "Surname": "Mustermann",
+        "Title": "Dr.",
+        "Role": "Musterrolle",
+        "Picture": true,
+        "Contacts": {
+            "Mobile": "+49150000000",
+            "Fax": "+49999999999",
+            "Email": "Max.Mustermann@example.com",
+            "Phone": "+493512820 0000",
```
Comment (paule96) [inline Phonebook.Source.MockPeopleBackend/mocks/real/persons.json]: please change this number to a more random one... This is to much bound to a company....

File: .azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+trigger: none
```
Comment (paule96) [inline .azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml]: maybe we should trigger a build for the mock backend to check if it's healthy.

File: Phonebook.Source.MockPeopleBackend/nginx/mime.types [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+types {
+    application/json                                 json;
```
Comment (paule96) [inline Phonebook.Source.MockPeopleBackend/nginx/mime.types]: why are here too much whitespace?

File: Phonebook.Source.MockPeopleBackend/real.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.17.0-alpine
+WORKDIR /etc/nginx
+
+RUN rm -r ./*
+COPY ./nginx/ ./
+
+
```
Comment (paule96) [inline Phonebook.Source.MockPeopleBackend/real.Dockerfile]: remove this empty line. To we have consistent one white line for regions.

File: Phonebook.Source.MockPeopleBackend/synthetic.Dockerfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.17.0-alpine
+WORKDIR /etc/nginx
+
+RUN rm -r ./*
+COPY ./nginx/ ./
+
+
```
Comment (paule96) [inline Phonebook.Source.MockPeopleBackend/synthetic.Dockerfile]: same here

File: .azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml (hunk 2 above)
Comment (DanielHabenicht) [reply .azure/pipelines/pr/Phonebook.Source.MockPeopleBackend.pr.yml]: There is indeed a PR Trigger for the PR right beneath: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#pr-triggers

File: Phonebook.Source.MockPeopleBackend/mocks/real/persons.json (hunk 1 above)
Comment (DanielHabenicht) [reply Phonebook.Source.MockPeopleBackend/mocks/real/persons.json]: done

**Comment by DanielHabenicht** [comment]
@paule96 updated with your recommendations

**Pull request merged**